src/streams/readAll.ts

```diff
@@ -27,6 +27,13 @@ export interface ReadAllOptions extends BaseOptions {
    * @default START
    */
   fromPosition?: ReadPosition;
+  /**
+   * The best way to explain link resolution is when using system projections. When reading the stream `$streams` (which
+   * contains all streams), each event is actually a link pointing to the first event of a stream. By enabling link
+   * resolution feature, the server will also return the event targeted by the link.
+   * @default false
+   */
+  resolveLinkTos?: boolean;
```

Comment:
**SUGGESTION:** I think that we should also add some tests to check if the resolution works correctly.